opflexagent/utils/ep_managers/endpoint_file_manager.py

```diff
@@ -606,9 +606,10 @@ def _map_dhcp_info(self, fixed_ips, mapping, mapping_dict):
                 dhcp4['lease-time'] = mapping['dhcp_lease_time']
             mapping_dict['dhcp4'] = dhcp4
             break
-        if len(v6subnets) > 0 and v6subnets.values()[0]['dns_nameservers']:
+        if len(v6subnets) > 0 and list(v6subnets.values())[0][
```

Comment:
Wonder why do we need to put list() around those returned by values() API here and below.....The values() API should already return a list even in python3 according to this,
https://www.tutorialspoint.com/python3/python_dictionary.htm